packages/ra-core/src/controller/input/useReferenceArrayInputController.ts

```diff
@@ -28,12 +28,14 @@ import { useResourceContext } from '../../core';
  *      source: 'referenceIds';
  * });
  *
- * @param {Object} option
```

Comment:
this change isn't necessary

Reply:
Are you sure? There is no param named `option` should be at least `* @param {Object} props`

Reply:
Done!